src/main/java/sirius/biz/storage/layer2/variants/ConversionEvent.java

```diff
@@ -0,0 +1,179 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.storage.layer2.variants;
+
+import sirius.biz.analytics.events.Event;
+import sirius.db.mixing.Mapping;
+import sirius.db.mixing.annotations.NullAllowed;
+
+/**
+ * Records the successful/failed conversion of a {@link BlobVariant} by a {@link Converter}.
+ */
+public class ConversionEvent extends Event {
+
+    /**
+     * Contains the database ID (if available) of the file used as a source for the conversion.
+     */
+    public static final Mapping SOURCE_ID = Mapping.named("sourceId");
+    @NullAllowed
+    private String sourceId;
+
+    /**
+     * Contains the file name of the file used as a source for the conversion.
+     */
+    public static final Mapping SOURCE_FILE_NAME = Mapping.named("sourceFileName");
+    private String sourceFileName;
+
+    /**
+     * Contains the file type / extension of the file used as a source for the conversion.
+     */
+    public static final Mapping SOURCE_TYPE = Mapping.named("sourceType");
+    private String sourceType;
+
+    /**
+     * Contains the database ID (if available) of the file used as the target or destination for the conversion.
+     */
+    public static final Mapping TARGET_ID = Mapping.named("targetId");
+    @NullAllowed
+    private String targetId;
+
+    /**
+     * Contains the name of the type / variant being created during conversion.
+     */
+    public static final Mapping TARGET_TYPE = Mapping.named("targetType");
+    private String targetType;
+
+    /**
+     * Contains the key of the {@link Converter} running the conversion.
+     */
+    public static final Mapping CONVERTER = Mapping.named("converter");
+    @NullAllowed
+    private String converter;
+
+    /**
+     * Contains the name of the actual tool / application used for the conversion.
+     */
+    public static final Mapping TOOL_NAME = Mapping.named("toolName");
+    @NullAllowed
+    private String toolName;
+
+    /**
+     * Contains whether or not the conversion resulted in the desired output file.
+     */
+    public static final Mapping SUCCESSFUL = Mapping.named("successful");
+    private boolean successful;
+
+    /**
+     * Contains an optional descriptive reason when the conversion failed.
+     */
+    public static final Mapping FAIL_REASON = Mapping.named("failReason");
+    @NullAllowed
+    private String failReason;
+
+    /**
+     * Contains how long the conversion took in milliseconds.
+     */
+    public static final Mapping DURATION = Mapping.named("duration");
+    private long duration = 0;
+
+    /**
+     * Contains the tenant owning the source and target files being converted.
+     */
+    public static final Mapping TENANT_ID = Mapping.named("tenantId");
+    @NullAllowed
+    private String tenantId;
```

Comment:
in the other 2 events with tenants already defined in biz, this field is simply called "tenant"  - unify these?